password-instances/password-instances.cabal

```diff
@@ -38,7 +38,7 @@ library
       base          >= 4.9 && < 5
     , aeson         >= 0.2
     , http-api-data
-    , password      >= 2
+    , password-types
```

Comment:
I hadn't actually thought about this, but it is neat that `password-instances` doesn't actually depend on `password` directly (but instead on `password-types`).